webapp/pages/settings/blocked-users.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div>
+    <ds-space>
+      <ds-card :header="$t('settings.blocked-users.name')">
+        <ds-text>
+          {{ $t('settings.blocked-users.explanation.intro') }}
+        </ds-text>
+        <ds-list>
+          <ds-list-item>
+            {{ $t('settings.blocked-users.explanation.your-perspective') }}
+          </ds-list-item>
+          <ds-list-item>
+            {{ $t('settings.blocked-users.explanation.their-perspective') }}
+          </ds-list-item>
+          <ds-list-item>
+            {{ $t('settings.blocked-users.explanation.notifications') }}
+          </ds-list-item>
+        </ds-list>
+      </ds-card>
+    </ds-space>
+    <ds-card v-if="blockedUsers && blockedUsers.length">
+      <ds-table :data="blockedUsers" :fields="fields" condensed>
+        <template #avatar="scope">
+          <nuxt-link
```

Comment:
Another moment to hate our styleguide. It is not possible to create a `blocked-user-table-row` component from it :angry:

Reply:
we'll refactor it out... `ReportRow` is a pretty good example, I think...

Reply:
sooo many styleguide elements! :scream:

but I guess we'll get rid of them one by one within the next few months... :grin: